docs/nb/bulk_validation_referential_integrity_check.ipynb

```diff
@@ -557,99 +591,76 @@
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 24,
+   "execution_count": 39,
    "id": "33516e3c-f10d-4c30-942b-0d01d06082f9",
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "['metagenome_sequencing_activity_set doc nmdc:wfmsa-11-0x1x5z82.1: field part_of referenced doc nmdc:wfmsa-11-0x1x5z82.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-wdgq0r08.1: field part_of referenced doc nmdc:wfmsa-11-wdgq0r08.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-20naj127.1: field part_of referenced doc nmdc:wfmsa-11-20naj127.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-wgr9bm58.1: field part_of referenced doc nmdc:wfmsa-11-wgr9bm58.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-g36rmj34.1: field part_of referenced doc nmdc:wfmsa-11-g36rmj34.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-eywfx790.1: field part_of referenced doc nmdc:wfmsa-11-eywfx790.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-h086zb46.1: field part_of referenced doc nmdc:wfmsa-11-h086zb46.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-ekgb9095.1: field part_of referenced doc nmdc:wfmsa-11-ekgb9095.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-hev7g094.1: field part_of referenced doc nmdc:wfmsa-11-hev7g094.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-85d13r42.1: field part_of referenced doc nmdc:wfmsa-11-85d13r42.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-g406y635.1: field part_of referenced doc nmdc:wfmsa-11-g406y635.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-ksg8wr62.1: field part_of referenced doc nmdc:wfmsa-11-ksg8wr62.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-f4v0tb15.1: field part_of referenced doc nmdc:wfmsa-11-f4v0tb15.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-103cks12.1: field part_of referenced doc nmdc:wfmsa-11-103cks12.1 not of type NamedThing',\n",
-       " 'metagenome_sequencing_activity_set doc nmdc:wfmsa-11-e0061a54.1: field part_of referenced doc nmdc:wfmsa-11-e0061a54.1 not of type NamedThing']"
-      ]
-     },
-     "execution_count": 24,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'id': 'nmdc:dobj-11-1epz0d53', 'id_is_nmdc_id': True, 'field': 'was_generated_by', 'value': 'nmdc:omprc-11-sxze4w22', 'slot_range': 'Activity'}\n",
```

Comment:
Activity here should be changed to DataGeneration

Reply:
I created issue https://github.com/microbiomedata/nmdc-runtime/issues/628 about this.